materialize-mysql/sqlgen.go

```diff
@@ -51,10 +57,10 @@ var mysqlDialect = func(tzLocation *time.Location, database string) sql.Dialect
 			),
 			sql.NUMBER:   sql.MapStatic("DOUBLE PRECISION", sql.AlsoCompatibleWith("double")),
 			sql.BOOLEAN:  sql.MapStatic("BOOLEAN", sql.AlsoCompatibleWith("tinyint")),
-			sql.OBJECT:   sql.MapStatic("JSON"),
-			sql.ARRAY:    sql.MapStatic("JSON"),
+			sql.OBJECT:   sql.MapStatic(jsonType),
```

Comment:
Should we use the `jsonMapper` for object and array flat types as well? It seems to me like we should, so that objects and arrays materialized as stringified JSON in MariaDB in the same way as multiples type fields.

Reply:
@williamhbaker I think it is not necessary, since OBJECT and ARRAY types will arrive as `json.RawMessage`, so jsonMapper is effectively no-op for these. Postgres materialization also has a similar mapping as mysql and it is working fine 🤔